gulpfile.js

```diff
@@ -318,6 +318,17 @@ gulp.task('connect', ['watch'], function() {
     res.set('Content-Type', 'application/javascript');
     res.send(configJs);
   });
+  //TODO check up with jdyson to figure out how this works on prod
+  hawtio.use('/apiman/config.js', function(req, res, next) {
+    var configJs = 'window.APIMAN_CONFIG = {' +
+      ' auth: {' +
```

Comment:
I assume you meant to use the APIMAN_CONFIG content in here rather than OS auth config?